### PR TITLE
feat: Add stickers

### PR DESCRIPTION
**Pull request by Kondrahin**
# Description

Added methods for interacting with sticker pack/stickers - `get_sticker_pack_list`, `get_sticker_pack`. `get_sticker_from_pack`, `create_sticker_pack`, `add_sticker`, `edit_sticker_pack`, `delete_sticker_pack`, `delete_sticker`.

# Checklist

- [x] Linters and tests have passed
- [x] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

File: botx/bots/mixins/requests/stickers.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        credentials: SendingCredentials,
+        name: str,
+        user_huid: UUID,
+    ) -> StickerPack:
+        """Create sticker pack.
+
+        Arguments:
+            credentials: credentials for making request.
+            name: name of sticker pack.
+            user_huid: id of sticker pack author.
```
Comment (Nimond) [inline botx/bots/mixins/requests/stickers.py:104]: Make arg descriptions similar between each other, `HUID of author`

File: botx/clients/methods/errors/stickers/image_not_valid.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""Definition for "image not valid" error."""
+from typing import NoReturn
+
+from botx.clients.methods.base import APIErrorResponse, BotXMethod
+from botx.clients.types.http import HTTPResponse
+from botx.exceptions import BotXAPIError
+
+
+class ImageNotValidError(BotXAPIError):
+    """Error for raising when image not valid."""
```
Comment (Nimond) [inline botx/clients/methods/errors/stickers/image_not_valid.py:10]: Maybe "Error for raising when image **is** not valid."?

File: botx/clients/methods/errors/stickers/image_not_valid.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    message_template = "image not valid"
```
Comment (Nimond) [inline botx/clients/methods/errors/stickers/image_not_valid.py:12]: The same

File: botx/clients/methods/errors/stickers/image_not_valid.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    """Handle "image not valid" error response.
+
+    Arguments:
+        method: method which was made before error.
+        response: HTTP response from BotX API.
+
+    Raises:
+        ImageNotValidError: raised always.
+    """
+    APIErrorResponse[dict].parse_obj(response.json_body)
```
Comment (Nimond) [inline botx/clients/methods/errors/stickers/image_not_valid.py:25]: For what?

File: botx/clients/methods/errors/stickers/sticker_pack_not_found.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+from botx.clients.types.http import HTTPResponse
+from botx.exceptions import BotXAPIError
+
+
+class StickerPackNotFoundError(BotXAPIError):
+    """Error for raising when sticker pack not found."""
+
+    message_template = "sticker pack {pack_id} not found"
+
+    #: ID of sticker pack that was requested.
```
Comment (Nimond) [inline botx/clients/methods/errors/stickers/sticker_pack_not_found.py:17]: From `mixins/requests/stickers.py`: `id of sticker pack`. Make docstrings the same.

File: botx/clients/methods/v3/stickers/sticker_pack_list.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+    __url__ = "/api/v3/botx/stickers/packs"
+    __method__ = "GET"
+    __returning__ = StickerPackList
+
+    #: Author ID.
+    user_huid: Optional[UUID]
+
+    #: Count of returning value.
+    limit: int = 1
```
Comment (Nimond) [inline botx/clients/methods/v3/stickers/sticker_pack_list.py:21]: You have already defined `1` as default in `StickersMixin.get_sticker_pack_list`

**Review by Nimond — APPROVED** [review]

**Review by vonabarak — APPROVED** [review]

**Pull request merged**